zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/autoconfigure/ObjectManagementAutoConfiguration.kt

```diff
@@ -69,6 +70,7 @@ class ObjectManagementAutoConfiguration {
         )
     }
 
+    @Primary
```

Comment:
I don't like having primary beans in the valtimo because you can't easily override them. Why is this required here?

Reply:
Changed